tasks/Module 4/task_4.py

```diff
@@ -0,0 +1,18 @@
+"""Spreadsheet Cell Inventer."""
+import openpyxl
+
+
+def spreadsheet_cell_inverter(file_path):
+    """Spreadsheet Cell Inventer."""
+    source_wb = openpyxl.load_workbook(file_path)
+    source_ws = source_wb.active
+    rotated_rows = list(zip(*reversed(list(source_ws.values))))
```

Comment:
This is smart :) But it when I run this code on machine I have headers (ITEM and SOLD) on the right side instead of on the left. I think it is because `reversed`. When you remove it you can as well skip converting `source_ws.values` generator to list, so this line can be just: `rotated_rows = list(zip(*source_ws.values))`.

Reply:
yes, you are right, my code rotates 2 dimension list to the right, when i need to reflect it by middle line